make-common.sh

```diff
@@ -21,8 +21,8 @@ while [ "$1" != "" ]; do
     debug)
       BUILDTYPE='Debug'
       ;;
-    macos-no-universal)
```

Comment:
i think we better keep this `macos-no-universal` option for compatibility reason, so dont remove it.

Reply:
ok, how do you want it to be?
1. by default build non-universal, but have options for both universal and non-universal
2. keep it as before - by default build universal, but have an option for non-universal

Reply:
1 is good to me.